docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3'
+networks:
+  db-netw:
+    driver: bridge
+services:
+  pythonapp:
+    image: app:1.0
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    networks:
+      - db-netw
+    depends_on:
+      - mysql
+    restart: unless-stopped
+  mysql: 
+    image: mysql-local:1.0.0
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: my-sql
+    ports:
+      - "3306:3306"
+    volumes: 
+      - db-volume:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_DATABASE=app_db
+      - MYSQL_USER=app_user
+      - MYSQL_PASSWORD=1234
+    networks:
+      - db-netw
+volumes:
+  db-volume:
```

Comment:
![image](https://github.com/mate-academy/devops_todolist_docker_core_task_3_docker_compose/assets/87720583/0b80de2b-a999-4465-b32e-9e7695982a08)
Add Eof(End-of-file), need to add empty line at the end